OOPAO/tools/tools.py

```diff
@@ -313,15 +313,22 @@ def compute_fourier_mode(pupil,spatial_frequency,angle_deg,zeropadding = 2):
     
     return mode
 
-#%%
-def circularProfile(img):
+
+def circularProfile(img, maximum = False):
     # Compute circular average profile from an image, reference to center of image
     # Get image parameters
     a = img.shape[0]
     b = img.shape[1]
     # Image center
     cen_x = a//2
     cen_y = b//2
+
+    if maximum == True:
+        cogMaximum = cog(img)
```

Comment:
cog should be changed to centroid() available in the same file. to be backward compatible I would suggest the following modification: 
```suggestion
        cogMaximum = centroid(img,threshold = 0.1*img.max())
```
The centroid function can be seen line 230

The import line 18 should be removed as this module is not part of OOPAO